specs/_features/maxeb_increase/capella.py

```diff
@@ -1162,8 +1194,11 @@ def slash_validator(state: BeaconState,
     Slash the validator with index ``slashed_index``.
     """
     epoch = get_current_epoch(state)
-    initiate_validator_exit(state, slashed_index)
+    cancel_validator_withdrawals(state, slashed_index)
     validator = state.validators[slashed_index]
+    # Reset the exit epoch for a new exit.
+    validator.exit_epoch = FAR_FUTURE_EPOCH
+    initiate_validator_exit(state, slashed_index)
     validator.slashed = True
     validator.withdrawable_epoch = max(validator.withdrawable_epoch, Epoch(epoch + EPOCHS_PER_SLASHINGS_VECTOR))
```

Comment:
This assumes that `validator.withdrawable_epoch` is set, which used to happen in `initiate_validator_exit`, but isn't the case here.

Reply:
i think this is resolved in the latest ?